application/testing/CMakeLists.txt

```diff
@@ -391,6 +391,7 @@ if(F3D_MODULE_RAYTRACING)
   f3d_test(NAME TestRaytracingDenoise DATA suzanne.ply THRESHOLD 120 ARGS -rd --samples=1)
   f3d_test(NAME TestVersionRaytracing ARGS --version REGEXP "Raytracing module: ON")
   f3d_test(NAME TestInteractionRaytracingDenoise DATA suzanne.ply THRESHOLD 120 ARGS --samples=1 INTERACTION) #RD
+  f3d_test(NAME TestRaytracingScalarBar DATA dragon.vtu ARGS -rsb --samples=1)
```

Comment:
could you add a TestDefaultConfigFileRaytracing and TestThumbnailConfigFileRaytracing ?